src/GraphQL.DataLoader.Tests/BatchDataLoaderTests.cs

```diff
@@ -383,4 +384,5 @@ public async Task Returns_Null_For_Null_Value_Types()
         var loader = new BatchDataLoader<int?, string>((_, _) => throw new Exception());
```

Comment:
This won't compile with NRT enabled because of the `where TKey : notnull` constraint.  So NRT is disabled here.